README.md

```diff
@@ -198,6 +198,12 @@ Successfully created compute instance ID  0 on GPU  1 GPU instance ID  2 using p
 
 You are now all set to dynamically create slices on the cluster using InstaSlice.
 
+## Deploy the cert manager
```

Comment:
References to the cert manager in this PR needs to be changed to instaslice.

Reply:
Thanks, the cert manager folder currently references the name as instaslicev2 since Kubebuilder was configured to call it. If you agree we can send a separate PR to change the reference from Instaslicev2 to Instaslice for the entire project.

Reply:
Makes sense. Thanks.